setup.py

```diff
@@ -37,49 +47,17 @@ def get_long_description():
     return fp.read()
 
 
-version = get_project_version()
-inclusive_min_tfx_version = version["INCLUSIVE_MIN_TFX_VERSION"]
-exclusive_max_tfx_version = version["EXCLUSIVE_MAX_TFX_VERSION"]
-
 TESTS_REQUIRE = ["pytest", "pylint", "pre-commit", "isort", "yapf"]
 
-required_tfx_version = "tfx>={},<{}".format(inclusive_min_tfx_version,
-                                            exclusive_max_tfx_version)
-required_ml_pipelines_sdk_version = "ml_pipelines_sdk>={},<{}".format(
-    inclusive_min_tfx_version, exclusive_max_tfx_version)
-required_ml_metadata_version = "ml_metadata>={},<{}".format(
-    inclusive_min_tfx_version, exclusive_max_tfx_version)
-
-PKG_REQUIRES = {
-    # Add dependencies here for your project. Avoid using install_requires.
-    "mlmd_client":
-    [required_ml_pipelines_sdk_version, required_ml_metadata_version],
-    "schema_curation": [
-        required_tfx_version,
-    ],
-    "feast_examplegen": [
-        required_tfx_version,
-        "feast>=0.16.0,<1.0.0",
-    ],
-    "xgboost_evaluator": [
-        required_tfx_version,
-        "xgboost>=1.0.0",
-    ],
-    "sampler": ["tensorflow>=2.0.0"],
-    "message_exit_handler": [
-        "kfp>=1.8,<1.9",
-        "slackclient>=2.9.0",
-        "pydantic>=1.8.0",
-    ],
-}
+PKG_REQUIRES = get_pkg_metadata()
 EXTRAS_REQUIRE = PKG_REQUIRES.copy()
 EXTRAS_REQUIRE["all"] = list(
     set(itertools.chain.from_iterable(list(PKG_REQUIRES.values()))))
 EXTRAS_REQUIRE["test"] = TESTS_REQUIRE
 
 setup(
     name=PROJECT_NAME,
-    version=version["__version__"],
+    version=get_version(),
```

Comment:
I'm not really clear on the need for this change, but I'm ok with it.  As long as `tfx_addons.__version__` returns the correct version I'm fine.

Reply:
yup. this is mostly housekeeping